lib/state/appState.js

```diff
@@ -89,6 +89,17 @@ appState.getInitial = function() {
   };
 };
 
+appState.hideUnavailableDevices = function() {
+  var currentOS = this.app.device._os; // "mac", "win", "android", "cros", "linux", or "openbsd"
```

Comment:
I agree with your reasoning, but I think the issue is that `chrome.runtime.getPlatformInfo` is asynchronous, so requires a callback. `App` is a React class, so I don't think we can use callbacks there (?), and `appState.js` doesn't use callbacks either. 

I can move it to `appActions.js`, but then we'll have to mock the `chrome` object, which would introduce a new dependency (`sinon-chrome`). More effort than it's worth?